bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/CTabFolder.java

```diff
@@ -774,6 +774,7 @@ void createItem (CTabItem item, int index) {
 	priority = newPriority;
 
 	if (items.length == 1) {
+		this.setSelection(0);
```

Comment:
Why use `this.` to qualify the call the method? That doesn't match the style of the rest of the code.

I looked at the implementation of what all happens from a `setSelection`.  It also calls `showItem` which calls
```
 updateFolder(REDRAW_TABS);
```
But that's got different arguments than the call below
```
updateFolder(UPDATE_TAB_HEIGHT | REDRAW);
```
That's probably okay because `updateFolder` does  `updateFlags |= flags` and defers actions to an async event.

Reply:
> Why use `this.` to qualify the call the method? That doesn't match the style of the rest of the code.

Do you mean this would be more matching in that context?
item.parent.setSelection(0);

It can be done this way as well.

Reply:
I think that `this.setSelection(0)` and `setSelection(0)` are the same, are they not?  Or asking it differently, would a different/wrong method be called if you deleted `this.`?

Reply:
> I think that `this.setSelection(0)` and `setSelection(0)` are the same, are they not? Or asking it differently, would a different/wrong method be called if you deleted `this.`?

setSelection(0);
This is also working the same way. You are right. 
I have updated and squashed all the commits. Thanks alot

Reply:
> It's really just nit picking, but keeping the style of the code consistent is a very good thing.

No problem atall, you are always welcome.

Reply:
> > It's really just nit picking, but keeping the style of the code consistent is a very good thing.
> 
> No problem atall, you are always welcome. It'll be a fine tuning for me :)

Reply:
Was `this` removed? I still see this.selection(0)